browser/ui/webui/new_tab_page/brave_new_tab_ui.cc

```diff
@@ -52,6 +53,17 @@
 
 using ntp_background_images::NTPCustomImagesSource;
 
+namespace {
+
+std::string GetDefaultSearchWidgetHost(PrefService* local_state) {
```

Comment:
Nit: this name is a little inconsistent IMO. Maybe `GetSearchWidgetDefault[Search]Host`?